auslib/AUS.py

```diff
@@ -99,7 +99,7 @@ def evaluateRules(self, updateQuery, transaction=None):
         # serve every request an update
         # backgroundRate=100 means all requests are served
         # backgroundRate=25 means only one quarter of requests are served
-        if not updateQuery["force"] == SUCCEED and rule["backgroundRate"] < 100:
+        if "force" in updateQuery and not updateQuery["force"] == SUCCEED and rule["backgroundRate"] < 100:
```

Comment:
There's no expectation that we'll be doing them for Guardian at this point. However, this makes me wonder if we should just support them in Balrog anyways, since we don't require any client changes to do so...

Reply:
Upon further reflection, I think supporting rollouts makes sense here for the reason above, as well as because it avoids yet another Rules column that Guardian doesn't use. I'll go ahead and add support, shouldn't be very difficult.